packages/moonstone/Spinner/Spinner.module.less

```diff
@@ -4,150 +4,99 @@
-	15%       { transform: rotateZ(160deg); }
-	25%       { transform: rotateZ(200deg); }
-	30%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
-	/* Impact at last ball */
-	33%       { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	39%, 67%  { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	/* Small bounce forward */
-	72%       { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	79%, 100% { transform: rotateZ(360 + @moon-spinner-ball-angle); }
+@keyframes rotate3 {
+	0%, 100% { transform: rotate(0turn); }
+	33% { transform: rotate((3turn / 8)); }
 }
-@keyframes spinBall2 {
-	/* Small bounce forward */
-	0%        { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	5%        { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	12%, 33%  { transform: rotateZ(360 + @moon-spinner-ball-angle); }
-
-	33.01%    { transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	48%       { transform: rotateZ(160deg); }
-	58%       { transform: rotateZ(200deg); }
-	63%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
-	/* Impact at last ball */
-	67%       { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	72%, 100% { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-}
-@keyframes spinBall3 {
-	/* Impact at last ball */
-	0%, 100%  { transform: rotateZ(360 - (@moon-spinner-ball-angle * 3)); }
-	5%, 33%   { transform: rotateZ(360 - @moon-spinner-ball-angle); }
-	/* Small bounce forward */
-	38%       { transform: rotateZ(360 + @moon-spinner-ball-angle + 10); }
-	45%, 66%  { transform: rotateZ(360 + @moon-spinner-ball-angle); }
-
-	66.01%    { transform: rotateZ(@moon-spinner-ball-angle); }
-	/* Slow down toward the top */
-	81%       { transform: rotateZ(160deg); }
-	91%       { transform: rotateZ(200deg); }
-	97%       { transform: rotateZ(225deg); }
-	/* Speed back up as the ball falls */
+@keyframes rotate4 {
+	0%, 100% { transform: rotate(0turn); }
+	33% { transform: rotate((4turn / 8)); }
 }
 
 .spinner {
 	line-height: @moon-body-line-height;
 
-	.ballDecorator {
-		border-radius: (@moon-spinner-size / 2);
+	.bg {
 		position: relative;
+		border-radius: 999px;
 		width: @moon-spinner-size;
 		height: @moon-spinner-size;
-		margin: 0 auto;
+		padding: @moon-spinner-line-margin;
+		margin-left: auto;
+		margin-right: auto;
 	}
 
-	.ball {
-		position: absolute;
-		text-align: center;
-		font-size: 90%;
-		line-height: 0.4em;
-		-webkit-animation: none @moon-spinner-time linear infinite;
+	.decorator {
+		width: @moon-spinner-size;
+		height: @moon-spinner-size;
+		position: relative;
+		background-size: cover;
+		border-radius: 999px;
+		overflow: hidden;
+		margin: 0 auto;
+		-webkit-mask-image: radial-gradient(transparent (@moon-spinner-size / 2 - @moon-spinner-line-width - 1px), black (@moon-spinner-size / 2 - @moon-spinner-line-width)); // These colors control masking, not visible colors
```

Comment:
We can use `mask-image` here and rely on PostCSS to prefix it for us. Will give us a little better cross-platform compatibility (though it appears Firefox supports the `webkit-` prefix and the unprefixed versions)

Reply:
Using the plain `mask-image` did not work in my testing. I can look into it further, but Even Chrome 73 preferred `-webkit-mask-image`.

Reply:
That's interesting. I didn't see the same in my testing. Well, Chrome still preferred the prefixed version but PostCSS output both and the result worked. Regardless :+1: for now